### PR TITLE
Pbind TableView support Auto-layout nested collectionView

### PR DESCRIPTION
**Pull request by renjialiang**
Hi Galen, 

在用Pbind时发现PBTableView使用动态CollectionView做为Cell布局时，没有根据动态的内容高度做自适应拉伸。

针对该问题对PBCollectionCell中的CollectionView设置了resizingDelegate代理；
并且对CollectionView重设frame时调用invalidateIntrinsicContentSize方法，使PBCollectionView获取正确的ContentSize。
         
并在Example中添加了Nested Table的使用例子

### CODE REVIEW

File: Pbind/Classes/View/PBCollectionView.h [hunk 1]
```diff
@@ -101,4 +101,6 @@
  */
 @property (nonatomic, strong) NSArray<id> *selectedDatas;
 
+
```
Comment (galenlin) [inline Pbind/Classes/View/PBCollectionView.h:104]: Please revert these unnecessary changes.

File: Pbind/Classes/View/PBCollectionView.m [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #import "PBSectionMapper.h"
 #import "UIView+Pbind.h"
 #import "PBCollectionViewFlowLayout.h"
-
+#import "PBTableView.h"
```
Comment (galenlin) [inline Pbind/Classes/View/PBCollectionView.m]: Please revert this unnecessary changes.

**Review by galenlin — COMMENTED** [review]
Thank you guy, you do me a BIG favor! 👍 

Note that there are some unnecessary changes, please revert them and re-commit. Then I will pick this PR ASAP. 🤝

**Comment by renjialiang** [comment]
Hi Galen,
我已经去掉无效引用和变动了，也感受到了你的严谨。

**Comment by galenlin** [comment]
Accepted.
